Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -81,10 +82,10 @@ protected class HandPanData
         [Tooltip("Each object listed must have a script that implements the IHandPanHandler interface or it will not receive events")]
         private GameObject[] panEventReceivers = null;
 
-        [Header("Geometry")]
+        [Header("Visual affordance")]
         [SerializeField]
         [Tooltip("If affordace geometry is desired to emphasize the touch points(leftPoint and rightPoint) and the center point between them (reticle), assign them here.")]
-        private GameObject reticle = null;
+        private GameObject centerPoint = null;
```

Comment:
The [FormerlySerializedAsAttribute](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html) should be added here, to preserve previous asset serialization.

Reply:
Added it. Thanks!